### ISSUE TITLE
Extension installation fails

### ISSUE THREAD

**Issue by panuhorsmalahti** [kontena/lens#7240]
How to reproduce:
1. Follow instructions at https://github.com/nevalla/lens-resource-map-extension#installation to install extension

What should happen: extension is installed.

What actually happens:
* Lens is stuck on "Installing extension" state
* Error logs:

```
npacking extension @nevalla/kube-resource-map@1.1.0 {fileName: 'kube-resource-map-1.1.0.tgz', tempFile: '/var/folders/_j/ztr5t9y90jq_jpsz1s2ytvk40000gp/T/lens-extensions/kube-resource-map-1.1.0.tgz'}
renderer.js:26389 Uncaught Error: [ExtensionInstallationStore]: cannot set /Users/phorsmalahti/.k8slens/extensions/nevalla--kube-resource-map/package.json as installing. Is currently installing.
    at ExtensionInstallationStateStore.value [as setInstalling] (renderer.js:26389:1)
    at EventEmitter.<anonymous> (renderer.js:26369:1)
    at EventEmitter.emit (node:events:526:28)
    at Object.onMessage (node:electron/js2c/renderer_init:69:755)
value @ renderer.js:26389
(anonymous) @ renderer.js:26369
emit @ node:events:526
onMessage @ node:electron/js2c/renderer_init:69
renderer.js:26472 Uncaught Error: [ExtensionInstallationStore]: cannot clear INSTALLING state for /Users/phorsmalahti/.k8slens/extensions/nevalla--kube-resource-map/package.json, it is currently idle
    at ExtensionInstallationStateStore.value [as clearInstalling] (renderer.js:26472:1)
    at EventEmitter.<anonymous> (renderer.js:26372:1)
    at EventEmitter.emit (node:events:526:28)
    at Object.onMessage (node:electron/js2c/renderer_init:69:755)
```

**Comment by panuhorsmalahti** [comment]
On `npm run dev` I got the same errors _but_  the extension was installed successfully.

EDIT: On another try on the latest beta the install was stuck but I didn't see any errors.

**Comment by jim-docker** [comment]
Is this a regression of #7141? Or the fix didn't make it in 6.4? Anyway, the fix here (#7247) is probably better

**Comment by Nokel81** [comment]
Might be a partial regression of #6715 as well.
